### DISCUSSION TITLE
Put all syntaxes used in the document to the top of the list, or allow for favorites.

### DISCUSSION THREAD

**Discussion by vsviridov** [outline/outline#7211] [Ideas]
Right now I write a bunch of documentation where I alternate between `bash` and `typescript`/`jsx` syntax, so every time I start a new block I have to hunt for them in the dropdown.

It would be pretty neat if all syntaxes used in the document so far would inform the dropdown and put them at the top of the list.

**Comment by tommoor** [comment]
This is a good idea, similar to emoji picker, probably track the 5 most recent languages and sort them to the top above a divider.
